slick.grid.js

```diff
@@ -388,7 +388,6 @@
       updatePinnedState();
       setupColumnSort();
       resizeCanvas();
-      updateAntiscroll();

```

Comment:
I removed this since I've moved the updateAntiscroll call to run in the resizeCanvas function itself.